lib/deobfuscate.py

```diff
@@ -7,6 +10,20 @@
 DEFAULT_MAX_DEPTH = 150
 
 
+def parse_base64_payload(payload: str) -> str:
+    if -1 == payload.find(r"Command/Base64/"):
+        return payload
+
+    logger.debug(f"Base64 command has been identify")
+    payload_without_base64, base64_command = payload.split(r"Command/Base64/")
```

Comment:
If the string `Command/Base64/` is embedded in the payload, the string will be split into more than 2 parts, so an exception will occur. 
```suggestion
    payload_without_base64, base64_command = payload.split("Command/Base64/", 1)
```

Reply:
Fixed